tools/accuracy_checker/accuracy_checker/metrics/metric_profiler/base_profiler.py

```diff
@@ -119,13 +134,15 @@ def write_json_result(self):
                     'device': self.device,
                     'tags': self.tags
                 },
-                'report': self.storage,
+                'report': list(self.storage.values()),
                 'report_type': self.__provider__,
                 'dataset_meta': self.dataset_meta
             }
         with open(str(out_path), 'w') as f:
             json.dump(out_dict, f)
 
+        self.storage = OrderedDict()
```

Comment:
possibly, it makes sense to create method for reset storage. if in future storage is changed, it will be easier maintain